scripts/webpack/webpack.dev.js

```diff
@@ -40,7 +40,6 @@ module.exports = (env = {}) => {
       rules: [
         {
           test: /\.tsx?$/,
-          exclude: /node_modules/,
```

Comment:
Webpack now sees the `@grafana/foo` packages coming from node_modules (they're symlinked there), so they need to be transpiled.